src/main/java/com/criteo/kafka/IncludeRegexMetricPredicate.java

```diff
@@ -0,0 +1,31 @@
+package com.criteo.kafka;
```

Comment:
this class is a duplicate of ExcludeRegexMetricPredicate, maybe you should modify RegexMetricPredicate class to allow to include or exclude at construction time.